system/lib/compiler-rt/lib/lsan/lsan_interceptors.cpp

```diff
@@ -453,17 +453,32 @@ INTERCEPTOR(int, pthread_create, void *th, void *attr,
   ENSURE_LSAN_INITED;
   EnsureMainThreadIDIsCorrect();
 
+#if !SANITIZER_EMSCRIPTEN
```

Comment:
I wonder if could just have a single `if` at the top of this function?  

```
#if !SANITIZER_EMSCRIPTEN
// Treat __ATTRP_C11_THREAD like the null attr
if (attr == __ATTRP_C11_THREAD) attr = nullptr;
#endif
```

Reply:
That's a good idea. Btw you meant `#if SANITIZER_EMSCRIPTEN` (without `!`), right?

Reply:
Yup